### ISSUE TITLE
Dox error in Holder description

### ISSUE THREAD

**Issue by rrahn** [seqan/seqan#1002]
The dox of http://docs.seqan.de/seqan/develop/?p=Holder#Holder%23assignValue
has a wrong link in its description

```
object	An object that holds a value or points to a value. Types: Holder, Iter Concepts: Concept.BasicOutputIteratorConcept
value	A value that is assigned to the item object holds or points to.
```

**Comment by xenigmax** [comment]
If you put below two lines after here (https://github.com/seqan/seqan/blob/develop/util/py_lib/seqan/dox/write_html.py#L273), it works properly for that particular link.
It may treats "#" in "See section" as a delimiter in a certain step (which is wrong).

```
if target == "assign":
    target = "AssignableConcept#assign"
```

**Comment by rrahn** [comment]
Well it seems that https://github.com/seqan/seqan/blob/develop/util/py_lib/seqan/dox/write_html.py#L274
produces the error, as the link it is processing is not part of the entries.
Not sure where it all comes from, but I think I get closer :tired_face:

**Comment by rrahn** [comment]
Fixed by #1128 and #1129.
